src/main/java/com/hierynomus/smbj/share/NamedPipe.java

```diff
@@ -125,14 +175,19 @@ public int read(byte[] buffer, int offset, int length) {
      * @return the number of bytes written to <code>outBuffer</code>
      */
     public int transact(byte[] inBuffer, byte[] outBuffer) {
-        return transact(inBuffer, 0, inBuffer.length, outBuffer, 0, outBuffer.length);
```

Comment:
Why aren't we calling the method below anymore? Code is exactly the same except for the range-check which is a no-op when called from this method.